React/Views/RCTSliderManager.m

```diff
@@ -104,7 +104,7 @@ - (void)slider:(id)slider didPress:(BOOL)press
 RCT_EXPORT_VIEW_PROPERTY(maximumTrackTintColor, UIColor);
 RCT_EXPORT_VIEW_PROPERTY(onValueChange, RCTBubblingEventBlock);
 RCT_EXPORT_VIEW_PROPERTY(onSlidingComplete, RCTDirectEventBlock);
-RCT_EXPORT_VIEW_PROPERTY(thumbTintColor, UIColor);
+RCT_EXPORT_NOT_OSX_VIEW_PROPERTY(thumbTintColor, UIColor); // TODO(macOS ISS#2323203)
```

Comment:
No such selector on NSSlider -- only on UISlider.   This would cause 'unknown selector' redbox errors on RNTester test pages and any App code that used <Slider> and this prop.